twingate/internal/utils/utils.go

```diff
@@ -22,3 +23,32 @@ func Filter[T any](items []T, ok func(item T) bool) []T {
 
 	return result
 }
+
+// Contains - checks if element exists in the slice.
+func Contains[T comparable](items []T, element T) bool {
+	lookup := MakeLookupMap[T](items)
+
+	return lookup[element]
+}
+
+// MapKeys - collects map keys to slice.
+func MapKeys[T comparable](lookup map[T]bool) []T {
```

Comment:
Maybe we should just include a `Set` implementation dependency?   would make the code nicer?